include/Utilities.h

```diff
@@ -25,12 +25,18 @@ namespace ChimeraTK::Utilities {
   std::string getPathName(const std::string& qualifiedName);
 
   /**
-   * Convert all characters which are not allowed in variable or module names into underscores.
-   * If allowDotsAndSlashes is true, dots and slashes will not be converted into underscores, so the resulting name can
-   * be a qualified name.
+   * Convert all characters which are not allowed in variable or module names into underscores followed by their
+   * 3-digit ascii code. An underscore will be escaped that way as well.
+   * If allowDotsAndSlashes is true, dots and slashes will not be converted, so the resulting name can be a qualified
+   * name.
    */
   std::string stripName(const std::string& name, bool allowDotsAndSlashes);
 
+  /**
+   * Undo the escaping done by stripName().
+   */
+  std::string unstripName(const std::string& name_stripped);
+
```

Comment:
That is odd naming. I'd go for escape/unescape or encode/decode, which is more common for this kind of operation. Stripping implies removing something, but that's not exactly what we do here

Reply:
You are right, technically it never striped anything. Maybe renaming is not a bad thing in any case if the behaviour changes.